Doc/using/cmdline.rst

```diff
@@ -316,12 +318,28 @@ Miscellaneous options
    * ``python -c code`` and ``python`` (REPL) command lines: Don't prepend an
      empty string, which means the current working directory.
 
-   See also the :envvar:`PYTHONSAFEPATH` environment variable, and :option:`-E`
+   See also the :envvar:`PYTHONSAFEPATH` environment variable, and :option:`-p`
    and :option:`-I` (isolated) options.
 
    .. versionadded:: 3.11
 
 
+.. cmdoption:: -p
+
+   Prepend a potentially unsafe path to :data:`sys.path`: opposite of the
+   :option:`-P` option.
+
+   This is the default Python behavior. This option can be used to ignore the
+   :option:`-P` command line option and the :envvar:`PYTHONSAFEPATH`
+   environment variable.
+
+   The :option:`-p` option takes precedence over the :option:`-P` and
+   :option:`-I` (isolated mode) options and the :envvar:`PYTHONSAFEPATH`
+   environment variable.
```

Comment:
I'd be concerned this might not be entirely clear that this doesn't override all of `-I`, just the `-P` it implies. What about something like this?

```suggestion
   The :option:`-p` option takes precedence over the :option:`-P` option
   (either passed explicitly or implied by :option:`-I`, isolated mode)
   and the :envvar:`PYTHONSAFEPATH` environment variable.
```

Reply:
Ola boa tarde